W3ChampionsStatisticService/Matches/OngoingMatchesCache.cs

```diff
@@ -38,24 +126,46 @@ public async Task<List<OnGoingMatchup>> LoadOnGoingMatches(
         int maxMmr,
         string sort)
```

Comment:
If we're changing things anyway, I think an enum would be better than using a string for this, but I dunno if we want to do that right now